2-mandatory/6-journey-planner.js

```diff
@@ -81,7 +83,9 @@ function getTransportModes() {}
 
   Hint: Use the corresponding array method to decide if an element is included in an array.
 */
-function isAccessibleByTransportMode() {}
+function isAccessibleByTransportMode(arr,mode) {
+  return arr.includes(`${mode}`);
+}
```

Comment:
Does array have an _includes_ method? I can't remember!
Is using string interpolation (backticks) redundant? Would this do the same thing ```return arr.includes(mode);```?